src/index.html

```diff
@@ -14,5 +14,38 @@
   </head>
   <body>
     <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-sat calendar--month-length-31">
```

Comment:
calendar should start from sunday
```suggestion
    <div class="calendar calendar--start-day-sun calendar--month-length-31">
```